### PR TITLE
Migration

### PR DESCRIPTION
**Pull request by alexanderleegs**
Merging Jekyll 4 migration changes to staging

### CODE REVIEW

File: _take-action/what-you-can-do.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Using cleaner transport options reduces our carbon footprint, and also keeps our
 - **[Support local produce](https://www.sfa.gov.sg/food-farming/sgfoodstory/supporting-local-produce){:class="external-link" target="_blank"}**  
 By supporting local produce, in addition to boosting Singapore’s food security and economy, consumers can also making environmentally sustainable choices in their everyday food consumption. Local produce is safe, fresher and lasts longer. Local produce also results in less spoilage, food waste, and a lower carbon footprint, as the produce does not need to travel for long periods of time before reaching the consumer.
 
-![Actionables](/resources/sgp_actionables.jpg)
+![Actionables](/files/resources/sgp_actionables.jpg)
```
Comment (gweiying) [inline _take-action/what-you-can-do.md:45]: typo

Comment (gweiying) [reply _take-action/what-you-can-do.md:45]: Also, in line 50 of this file, we could change the reference for them just in case they uncomment this

**Review by gweiying — CHANGES_REQUESTED** [review]
small changes

File: _take-action/what-you-can-do.md (hunk 1 above)
Comment (alexanderleegs) [reply _take-action/what-you-can-do.md]: Good catch, https://github.com/isomerpages/mse-sgp/pull/36/commits/1f6efc94f783d95a84f04681bbf1d003999854ef

**Review by gweiying — APPROVED** [review]
lgtm!

**Pull request merged**